cmd/gaia/app/app.go

```diff
@@ -183,6 +183,11 @@ func (app *GaiaApp) initChainer(ctx sdk.Context, req abci.RequestInitChain) abci
 
 	gov.InitGenesis(ctx, app.govKeeper, gov.DefaultGenesisState())
 
+	err = slashing.InitGenesis(ctx, app.slashingKeeper, genesisState.SlashingData)
```

Comment:
Not sure what's happening here, can someone explain via a comment in the review here?  I'll check it again next time I review this PR again.

Reply:
slashing.InitGenesis takes initial parameters from genesis file and stores them in the paramstore subspace.